caom2/caom2/checksum.py

```diff
@@ -309,31 +309,30 @@ def update_checksum(checksum, value, attribute=''):
             if not isinstance(value[i], AbstractCaomEntity):
                 checksums.append(value[i]._id)
         for i in sorted(checksums):
-            updated &= update_checksum(checksum, checksum[i], attribute)
+            updated |= update_checksum(checksum, checksum[i], attribute)
         return updated
     else:
         raise ValueError(
             'Cannot transform in bytes: {}({})'.format(value, type(value)))
 
     if b is not None:
         checksum.update(b)
-        if logger.isEnabledFor(logging.DEBUG):
-            logger.debug("Encoded attribute value - {} = {} {} bytes".format(attribute, value, len(b)))
+        logger.debug("Encoded attribute value - {} = {} {} bytes".format(attribute, value, len(b)))
         return True
     return False
 
 
-def to_model_name(attribute):
+def to_checksum_name(attribute):
     """
-    Converts the attribute name to the corresponding model name
+    Converts the attribute name to the corresponding model name in lower case
     :param attribute: name of attribute
     :return: camel case name of the attribute in the model
     """
     # Replace underscores and capitalize the first letter of each word
```

Comment:
No longer doing capitalization?

Reply:
All lowercase for reasons explained above.